### DISCUSSION TITLE
希望能增加form表单 以及form表单校验

### DISCUSSION THREAD

**Discussion by houqiangxie** [Moonofweisheng/wot-design-uni#60] [Ideas]
### 这个功能解决了什么问题？

根据表单rule，支持表单change,blur,submit时候校验必填属性，并暴露出api，支持自定义组件必填属性校验

### 你期望的 API 是什么样子的？

<uni-forms ref="dynamicForm" :rules="dynamicRules" :model="dynamicFormData">
	<uni-forms-item label="邮箱" required name="email">
		<uni-easyinput v-model="dynamicFormData.email" placeholder="请输入邮箱" />
	</uni-forms-item>
	<template v-for="(item,index) in dynamicFormData.domains">
		<uni-forms-item :label="item.label+' '+index" required
			:rules="[{'required': true,errorMessage: '域名项必填'}]" :key="item.id"
			:name="['domains',index,'value']">
			<view class="form-item">
				<uni-easyinput v-model="dynamicFormData.domains[index].value" placeholder="请输入域名" />
				<button class="button" size="mini" type="default" @click="del(item.id)">删除</button>
			</view>
		</uni-forms-item>
	</template>

</uni-forms>

**Comment by Moonofweisheng** [comment]
表单组件已计划新增，进度可见[讨论区](https://github.com/Moonofweisheng/wot-design-uni/discussions/45)
